specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,832 @@
+          "type": "string",
+          "description": "the region to create private link association."
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "x-nullable": true
+          }
+        }
+      }
+    },
+    "ResourceManagementPrivateLinkPatch": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "x-nullable": true
+          }
+        }
+      }
+    },
+    "ResourceManagementPrivateLinkEndpointConnections": {
+      "type": "object",
+      "properties": {
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The private endpoint connections."
+        }
+      }
+    },
+    "ResourceManagementPrivateLinkListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceManagementPrivateLink"
+          },
+          "description": "An array of resource management private links."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of resource information. Call ListNext() with this to fetch the next page of resource information."
+        }
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      },
+      "description": "An error response for a resource management request."
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for this operation.",
+      "x-ms-parameter-location": "client"
+    },
+    "GroupIdParameter": {
+      "name": "groupId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The management group ID.",
+      "x-ms-parameter-location": "method",
+      "minLength": 1,
+      "maxLength": 90
+    },
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "x-ms-parameter-location": "method"
+    },
+    "PrivateLinkAssociationIdParameter": {
+      "name": "plaId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the PLA",
+      "x-ms-parameter-location": "method"
+    },
+    "ResourceManagementPrivateLinkParameter": {
```

Comment:
>ResourceManagementPrivateLinkParameter [](http://example.com/codeflow?start=5&length=38)

is it a 'name' or just a 'parameter'? I think you can improve the naming!

Reply:
Keeping this as Parameter as changing the variable name now is a breaking change. This is the resource management private link name.